File: admoblibrary/src/main/java/com/rikatech/admoblibrary/AdsInterCallBack.java
package com.rikatech.admoblibrary;

import com.google.android.gms.ads.AdValue;

public interface AdsInterCallBack {
    void onStartAction();
    void onEventClickAdClosed();
    void onAdShowed();
    void onAdLoaded();
    void onAdFail(String error);
    void onPaid(AdValue adValue, String adUnitAds);
}

File: admoblibrary/src/main/java/com/rikatech/admoblibrary/callback/AdsInterCallBack.kt
package com.rikatech.admoblibrary.callback

import com.google.android.gms.ads.AdValue

interface AdsInterCallBack {
    fun onStartAction()
    fun onEventClickAdClosed()
    fun onAdShowed()
    fun onAdLoaded()
    fun onAdFail(error: String?)
    fun onPaid(adValue: AdValue, adUnitAds: String?)
}
